javascript/dwa-starter-vue/.tbd-example.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "dwa-starter-vue",
+  "tests": {
+    "pre": ["npm install", "npx playwright install --with-deps"],
+    "command": "npm run test:unit && npm run test:e2e"
```

Comment:
@Johnnyevans32 here you should be using `pnpm` instead of `npm`, exactly like you added in the `README.md`

Reply:
updated now